parentPom.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.exasol</groupId>
+    <artifactId>virtual-schema-common-document-parent</artifactId>
+    <packaging>pom</packaging>
+    <version>${revision}</version>
+    <properties>
+        <revision>6.1.0-SNAPSHOT</revision>
```

Comment:
We need to remember to remove the `-SNAPSHOT` ;)

Reply:
yes